src/commands/cluster-replicas.json

```diff
@@ -3,7 +3,7 @@
         "summary": "List replica nodes of the specified master node",
         "complexity": "O(1)",
         "group": "cluster",
-        "since": "5.0.0",
+        "since": "6.0.0",
```

Comment:
as far as i can tell from the source code in git, it was introduced in 5.0, not in 6.0.
where did you get the idea that it's wrong?
p.s. if i'm right, update the deprecation notice too

Reply:
Lesson learned: never trust gh's web ui. I'll revert and deprecate if you teach me the canonical git fu for looking up the first version a commit was included in :)

Reply:
i simply did `git show 5.0.0:src/cluster.c |grep \"replicas\"`.

walking on the git log for that purpose is no good since the cherry-picks are clones (not part of the graph).